ironmon_tracker/screens/ExtrasScreen.lua

```diff
@@ -9,7 +9,8 @@ ExtrasScreen = {
 		resultAboveAverage = "Above average!",
 		resultDecent = "Decent.",
 		resultUnavailable = "Estimate is unavailable.",
-		viewLogFile = "View the log",
+        viewLogFile = "View log",
```

Comment:
Looks like the spacing is using actual spaces here instead of tabs. Can you revert this back to using tabs like the line above and below it.